app/team.go

```diff
@@ -1127,6 +1127,45 @@ func (a *App) prepareInviteNewUsersToTeam(teamId, senderId string) (*model.User,
 	return user, team, nil
 }
 
+func genEmailInviteWithErrorList(emailList []string) []*model.EmailInviteWithError {
+	var invitesNotSent []*model.EmailInviteWithError
```

Comment:
nit: here we can optimize with `invitesNotSent := make([]*model.EmailInviteWithError, len(emailList))`